config/sync/block.block.help.yml

```diff
@@ -0,0 +1,20 @@
+uuid: 647a802d-5a41-48ce-a724-c849c1b7466e
```

Comment:
Is this block relevant to the PR? Not a problem if it isn't, especially given that it's for the admin theme.

Reply:
Right, yes. I added this block because I added some help to the Cohort node (cf. `config/sync/node.type.cohort.yml`) that wasn't being displayed on node create/edit forms. Hence switching on Help module and adding this block. 

Originally I used [serial](https://www.drupal.org/project/serial) contrib module so the cohort number was being incremented automatically by the system without need for human intervention. I then removed it because, of course, the 'cohort number' field will be synchronised with Salesforce in due course — Salesforce will define the number assigned to the cohort (presumably from a serial/auto-incrementing field there) so there's not the same need for this help to display. I can remove it, and switch off Help module.

Reply:
Done now, in be55f9a